docker/README.md

```diff
@@ -0,0 +1,13 @@
+# Containers
+
+This directory containers docker files for use with Orchid.
```

Comment:
@saurik I know you are generally a fan for three character directory prefixes. Where would you like to see the dockerfiles live?